tests/Content/FieldMethodsTest.php

```diff
@@ -1,1110 +0,0 @@
-<?php
-
-namespace Kirby\Content;
-
-use Kirby\Cms\App;
-use Kirby\Cms\Blocks;
-use Kirby\Cms\Collection;
-use Kirby\Cms\Files;
-use Kirby\Cms\Layouts;
-use Kirby\Cms\Page;
-use Kirby\Cms\Pages;
-use Kirby\Cms\Site;
-use Kirby\Cms\Users;
-use Kirby\Data\Data;
-use Kirby\Data\Json;
-use Kirby\Data\Yaml;
-use Kirby\Exception\InvalidArgumentException;
-use Kirby\Filesystem\Dir;
-use Kirby\Image\QrCode;
-use Kirby\TestCase;
-
-class FieldMethodsTest extends TestCase
```

Comment:
`Suggestion`: I actually think having the field methods tests in a separate test class still makes sense, especially given that the methods are defined in a trait.